thavalon-server/src/database/games/db_game.rs

```diff
@@ -212,15 +234,21 @@ impl DatabaseGame {
         }
 
         self.players.remove(player_id);
-        self.display_names.remove(display_name);
+        let display_name = match self.players_to_display_names.remove(player_id) {
+            Some(name) => name,
+            None => return Ok(None),
```

Comment:
should this log something? None should only occur if trying to remove a player not in game, right?